src/main/java/core/basesyntax/service/strategy/strategyimpl/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.strategy.strategyimpl;
+
+import core.basesyntax.dao.Storage;
+import core.basesyntax.model.FruitRecord;
+
+public class BalanceOperation implements OperationHandler {
```

Comment:
```suggestion
public class BalanceOperationHandler implements OperationHandler {
```
the same for other handlers